webpack.shared.js

```diff
@@ -9,8 +9,6 @@ const sharedDeps = [
     'react-i18next',
```

Comment:
I think it's better to migrate using `@jahia/webpack-config/getModuleFederationConfig` here, as it filters some of the modules listed in shared against the dependencies, so we don't need to manually maintain this.

Reply:
as per @r3dm1ke to be addressed on next guild day, but need the change after update to apollo-client